records/2023-Q1-roadmap.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: contact
+---
+
+This page lists the projects that the Scala Center will work on during
```

Comment:
I think we should use a present tense with `the current quarter`

```suggestion
This page lists the projects that the Scala Center are working on during
```

Reply:
I am not sure, maybe @ckipp01 could help us? :pray:

Reply:
Yup, I'd also recommend using `are working` since it's present tense.